roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -123,3 +123,56 @@
     volume_mounts: "{{ openshift_hosted_registry_volumes }}"
     edits: "{{ openshift_hosted_registry_edits }}"
     force: "{{ True|bool in openshift_hosted_registry_force }}"
+
+- block:
+  - name: Wait for registry pods
+    oc_obj:
+      namespace: "{{ openshift_hosted_registry_namespace }}"
+      state: list
+      kind: pod
+      selector: "{{ openshift_hosted_registry_name }}={{ openshift_hosted_registry_namespace }}"
+    register: registry_pods
+    until:
+    - "registry_pods.results.results[0]['items'] | count > 0"
+    # There must be as many matching pods with 'Ready' status True as there are expected replicas
+    - "registry_pods.results.results[0]['items'] | oo_collect(attribute='status.conditions') | oo_collect(attribute='status', filters={'type': 'Ready'}) | map('bool') | select | list | count == openshift_hosted_registry_replicas | int"
+    delay: 10
+    retries: "{{ (600 / 10) | int }}"
```

Comment:
I went with the hard-coded default specified [here](https://github.com/jarrpa/openshift-ansible/blob/b20075e050633445883e89860a5a1029b2fdb15a/roles/openshift_hosted/tasks/registry/registry.yml#L53). ;)